src/www/qserv/js/ReplicationController.js

```diff
@@ -148,22 +176,21 @@ function(CSSLoader,
                 this._loadFromScratch();
             });
         }
-        
-        /**
-         * Table for displaying the general status of the Master Replication Controller
-         * @returns JQuery table object
-         */
+        _status() {
```

Comment:
Yay!! Doxygen comments going away!  Extra bonus points -- I will buy you a beer! :-)